components/about/CustomTable.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class='officer-table'>
+    <tr v-for="(item, index) in listdata" :key="index">
+        <th>{{item.position}}</th>
+        <td>{{item.name}}</td>
+    </tr>
+    </div>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4 spaces.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7665075)
```suggestion
  </div>
```